exposed-services/charts/v1.0.0/exposed-service/templates/service.yaml

```
@@ -10,6 +10,9 @@ metadata:
   labels:
     greenhouse.sap/expose: "true"
     {{- include "exposed-service.labels" . | nindent 4 }}
+  annotations:
+    greenhouse.sap/expose: "true"
+    {{- include "exposed-service.serviceAnnotations" . | nindent 4 }}
```

In this case we can skip a dedicated template and chain the toYaml function inline
```suggestion
    {{- .Values.exposed-service.serviceAnnotations | toYaml | nindent 4 }}
```

---

I believe helper can be removed and it should be `{{- .Values.exposed-service.annotations | toYaml | nindent 4 }}`, right?